### PR TITLE
Issues 1

### PR DESCRIPTION
**Pull request by payamST**

### CODE REVIEW

File: magkit-test-servlet/.gitignore
Comment (serialcool) [inline magkit-test-servlet/.gitignore:1]: Just one gitignore is enough. Please remove this.

File: magkit-test-cms/pom.xml [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+
+    <artifactId>magkit-test-cms</artifactId>
+    <name>magkit-test-cms Magnolia Module</name>
+    <packaging>jar</packaging>
+
+    <properties>
+        <version.magkit-test-jcr>1.0.0-SNAPSHOT</version.magkit-test-jcr>
+        <version.magkit-test-servlet>1.0.0-SNAPSHOT</version.magkit-test-servlet>
+        <version.mockito-core>1.10.19</version.mockito-core>
+        <version.magnolia-core>6.2.17</version.magnolia-core>
```
Comment (serialcool) [inline magkit-test-cms/pom.xml:20]: Use version.magnolia property instead of new property of core module.

File: magkit-test-cms/pom.xml [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+    </parent>
+
+    <artifactId>magkit-test-cms</artifactId>
+    <name>magkit-test-cms Magnolia Module</name>
+    <packaging>jar</packaging>
+
+    <properties>
+        <version.magkit-test-jcr>1.0.0-SNAPSHOT</version.magkit-test-jcr>
+        <version.magkit-test-servlet>1.0.0-SNAPSHOT</version.magkit-test-servlet>
+        <version.mockito-core>1.10.19</version.mockito-core>
```
Comment (serialcool) [inline magkit-test-cms/pom.xml:19]: Try to use version from Magnolia's external BOM.

**Review by jfrantzius — APPROVED** [review]

File: magkit-test-cms/pom.xml (hunk 1 above)
Comment (serialcool) [reply magkit-test-cms/pom.xml:20]: If this to force downgrade of tests-Library, then we have to fix this later by upgrading used Mockito version.

File: magkit-test-jcr/pom.xml [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+    </parent>
+
+    <artifactId>magkit-test-jcr</artifactId>
+    <name>magkit-test-jcr Magnolia Module</name>
+    <description>Utility classes to easily mock javax.jcr classes (version 2.0).</description>
+    <packaging>jar</packaging>
+
+
+    <properties>
+        <version.commons-io>2.6</version.commons-io>
```
Comment (serialcool) [inline magkit-test-jcr/pom.xml:18]: try to use external BOM from Magnolia instead of define versions

**Review by wolf-bubenik — APPROVED** [review]

**Review by wolf-bubenik — APPROVED** [review]

**Pull request merged**